src/create-element.js

```diff
@@ -13,10 +13,7 @@ export function createElement(type, props, children) {
 	props = assign({}, props);
 
 	if (arguments.length>3) {
-		children = [children];
-		for (let i=3; i<arguments.length; i++) {
-			children.push(arguments[i]);
-		}
+		children = Array.prototype.slice.call(arguments, 2);
```

Comment:
Really like this one 👍 😄I feel like the initial version was stuff that got generated from a spread operator 🤔? Also, this comes with a nice `-29B` so I _really_ dig it!

Question: what if we drop the `if(arguments.length > 3)` completely? And just rely on:
```
children = Array.prototype.slice.call(arguments, 2);
```

Reply:
If we get rid of `if(arguments.length > 3)`, than we can change logic, because `children` will be an array always.

Reply:
If we have only 3 arguments, than we will be create additional array. We need to check if it hits performance.

Reply:
Ah, actually `children` could be something else than an `array`. I think it can't be dropped the `if` I mean.
Also, this actually shaves `-12B` which is still not bad 🎉!